packages/vscode-extension/src/extension.ts

```diff
@@ -1,28 +1,46 @@
+/*
+ ** Copyright (c) 2020 Oracle and/or its affiliates.  All rights reserved.
+ ** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+ */
+
 // The module 'vscode' contains the VS Code extensibility API
 // Import the module and reference it with the alias vscode in your code below
 import * as vscode from 'vscode';
-import deploy from './commands/Deploy';
-import listobjects from './commands/ListObjects';
+import { ActionExecutor } from './ActionExecutor';
+import AddDependencies from './commands/AddDependencies';
+import Deploy from './commands/Deploy';
+import ListObjects from './commands/ListObjects';
+import VSConsoleLogger from './loggers/VSConsoleLogger';
 const SCLOUD_OUTPUT_CHANNEL_NAME = 'Netsuite SuiteCloud';
 
-export const scloudOutput: vscode.OutputChannel = vscode.window.createOutputChannel(SCLOUD_OUTPUT_CHANNEL_NAME);
+export const Output: vscode.OutputChannel = vscode.window.createOutputChannel(SCLOUD_OUTPUT_CHANNEL_NAME);
 
 // this method is called when your extension is activated
 // your extension is activated the very first time the command is executed
 export function activate(context: vscode.ExtensionContext) {
 	// Use the console to output diagnostic information (console.log) and errors (console.error)
 	// This line of code will only be executed once when your extension is activated
-	console.log('Let\'s rock it, Netsuite SuiteCloud VSCode Extension has been activated!');
+	new VSConsoleLogger().info("Let's rock it, Netsuite SuiteCloud VSCode Extension has been activated!");
```

Comment:
We should not show this message in the SuiteCloud output channel. I was here like a internal comment while debugging but is not intended to show to the users. We can use the console.log as before.